### PR TITLE
Transition pages redirects for Sprint 11.6

### PR DESCRIPTION
**Pull request by kathycarothers**
Redirects for Sprint 11.6

### CODE REVIEW

File: redirects-transition.conf [hunk 1]
```diff
@@ -1,5 +1,12 @@
 #Redirect rules for specific pages
 
+
+rewrite ^/pages/brochures/pubfund_limits_2016.shtml https://www.fec.gov/help-candidates-and-committees/understanding-public-funding-presidential-elections/presidential-spending-limits-2016/ redirect;
+rewrite ^/pdf/record/1999/index99_1.htm https://www.fec.gov/resources/cms-content/documents/index1999.pdf redirect;
+rewrite ^/pdf/record/1998/index98_1.htm https://www.fec.gov/resources/cms-content/documents/index1998.pdf redirect;
+rewrite ^/pdf/record/1997/index97_1.htm https://www.fec.gov/resources/cms-content/documents/index1997.pdf redirect;
+rewrite ^/pages/brochures/jointfundraising_brochure.pdf https://www.fec.gov/help-candidates-and-committees/making-disbursements/joint-fundraising-candidates-political-committees/ redirect;
+rewrite ^/rad/documents/FECFileGettingStartedManual.pdf https://www.fec.gov/resources/cms-content/documents/FECFile_GettingStartedManual_Candidates.pdf
```
Comment (patphongs) [inline redirects-transition.conf]: @kathycarothers You're missing this ` redirect;` at the end of this redirect on line 9

Comment (dorothyyeager) [reply redirects-transition.conf]: Fixed now @patphongs

Comment (dorothyyeager) [reply redirects-transition.conf]: Added in a redirect suggested by https://github.com/fecgov/FEC/issues/9579

**Comment by pkfec** [comment]
@patphongs Looks like @dorothyyeager fixed the missing `redirect` on line#9. 
Could you please review this PR again?

**Pull request merged**